packages/react-core/src/demos/CardView/examples/CardView.md

```diff
@@ -549,39 +560,43 @@ class CardViewBasic extends React.Component {
     } = this.state;
 
     const toolbarKebabDropdownItems = [
-      <DropdownItem key="link">Link</DropdownItem>,
-      <DropdownItem key="action" component="button">
+      <OverflowMenuDropdownItem key="link">Link</OverflowMenuDropdownItem>,
+      <OverflowMenuDropdownItem key="action" component="button">
         Action
-      </DropdownItem>,
-      <DropdownItem key="disabled link" isDisabled>
+      </OverflowMenuDropdownItem>,
+      <OverflowMenuDropdownItem key="disabled link" isDisabled>
         Disabled Link
-      </DropdownItem>,
-      <DropdownItem key="disabled action" isDisabled component="button">
+      </OverflowMenuDropdownItem>,
+      <OverflowMenuDropdownItem key="disabled action" isDisabled component="button">
         Disabled Action
-      </DropdownItem>,
+      </OverflowMenuDropdownItem>,
       <DropdownSeparator key="separator" />,
-      <DropdownItem key="separated link">Separated Link</DropdownItem>,
-      <DropdownItem key="separated action" component="button">
+      <OverflowMenuDropdownItem key="separated link">Separated Link</OverflowMenuDropdownItem>,
+      <OverflowMenuDropdownItem key="separated action" component="button">
         Separated Action
-      </DropdownItem>
+      </OverflowMenuDropdownItem>
     ];
 
     const toolbarItems = (
       <React.Fragment>
         <ToolbarItem variant="bulk-select">{this.buildSelectDropdown()}</ToolbarItem>
-        <ToolbarItem>{this.buildFilterDropdown()}</ToolbarItem>
+        <ToolbarItem toggleIcon={<FilterIcon />} breakpoint="xl">{this.buildFilterDropdown()}</ToolbarItem>
         <ToolbarItem>
+        <OverflowMenu breakpoint="xl">
+        <OverflowMenuItem>
```

Comment:
Looks like indentation is off in this block

Reply:
will fix